projects/core/directives/dropdown/dropdown.component.ts

```diff
@@ -72,7 +72,9 @@ export class TuiDropdownComponent {
     protected readonly sub = inject(TuiPositionService)
         .pipe(
             takeWhile(
-                () => this.directive.el.isConnected && !!this.directive.el.offsetParent,
+                () =>
```

Comment:
could you add e2e test please?

Reply:
don't want to clutter up the demo, non target case